src/com/qamp/HarisJasarevic/homeworks/homework5/task1/SavingsAccount.java

```diff
@@ -0,0 +1,54 @@
+package com.qamp.HarisJasarevic.homeworks.homework5.task1;
+
+public class SavingsAccount extends BankAccounts {
+
+    private double dailyTransactionLimit;
+    private int transactionsLeft;
+
+    protected SavingsAccount (final AccountOwner accountOwner, final String serialNumber, final int passCode, final int dailyTransactionLimit) {
+        super(accountOwner, serialNumber, passCode);
+        this.dailyTransactionLimit = 1000;
+        this.transactionsLeft = 3;
+    }
+
+    @Override
+
+    public double moneyWithdraw (final double withdrawAmount) throws IllegalStateException {
+        if (balance - withdrawAmount < 0) {
+            throw new IllegalStateException("There is no enough funds available!");
+        }
+        if (withdrawAmount > dailyTransactionLimit) {
```

Comment:
Maybe I should have explained this a bit more, my idea was that if you withdraw 1st time 500 and then try 600 it should fail so daily in total you should not be able to exceed the limit.